lib/12T/ngspice/gf180mcu_osu_sc_12T_addf_1.ext

```diff
@@ -0,0 +1,181 @@
+timestamp 1660078482
+version 8.3
+.inc "/import/yukari1/lrburle/globalfoundries-pdk-libs-gf180mcu_osu_sc/char/techfiles/design.hspice"
```

Comment:
this seems to still reference absolute hostname/username?

Reply:
not really needed so deleted.  Thx!